src/PharoThings-Devices-HD44780.package/PotLCDHD44780I2C.class/instance/configureDefaultPeripherals.st

```diff
@@ -0,0 +1,3 @@
+initialization
+configureDefaultPeripherals
+	^ nil
```

Comment:
In smalltalk style it should be just an empty method (it returns self by default).
I saw it in other places too.

Reply:
Ok, I removed them